aws/aws-parallelcluster-cookbook
# Disable DLAMI aws-ubuntu-eni-helper to prevent confict with configure_nw_interface.sh

### Description of changes
* Disable aws-ubuntu-eni-helper service during image-build to avoid conflict with the script configure_nw_interface.sh that configure multiple nic on the instance
* Added kitchen test to verify the disabling

### Tests
* Manually tested on a DLAMI ubuntu OS
* Automated tests the next build cycle

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
